docs/app/Layouts/HomepageLayout.js

```diff
@@ -69,7 +69,7 @@ class DesktopContainer extends Component {
     const { fixed } = this.state
 
     return (
-      <Responsive {...Responsive.onlyComputer}>
+      <Responsive {...Responsive.onlyComputer} minWidth={768}>
```

Comment:
I'm not sure on this.  The props being spread in `Responsive.onlyComputer` include a minWidth of 992.  Why is is that we need to override this to 768?

https://github.com/Semantic-Org/Semantic-UI-React/blob/8ccee0f75fb2a9802bbaf33394182dc97e8b5d2e/src/addons/Responsive/Responsive.js#L56

If 768 is indeed correct, then we should not be spreading the other value and then overridding it.